examples/experimental/onnx/mobilenet_v2/utils.py

```diff
@@ -0,0 +1,100 @@
+# This code was taken as is from the pytorch repository. You
```

Comment:
Just for your information, it is better to keep samples minimalistic, e.g. AverageMeter, ProgressMeter, Accuracy are not required here. It is better to use standard things, such as tqdm, or sklearn, like:
```
from sklearn.metrics import accuracy_score
for item in tqdm(dataset):
    ...
    predictions.append(prediction_id)
    references.append(item["label"])

metric = accuracy_score(references, predictions)

```

Reply:
Thanks for the comment! Let's do it in a separate PR.